projects/sunbird-epub-player/src/lib/epub-viewer/epub-viewer.component.html

```diff
@@ -0,0 +1 @@
+<div id="area" #epubViewer></div>
```

Comment:
instead of **area** as id keep content id as a suffix to make it unique in a HTML doc